grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy

```diff
@@ -61,32 +62,32 @@ class InvoiceService {
         return invoiceItems
     }
 
-    def getInvoiceCandidates(String id, String orderNumber, String shipmentNumber, String max, String offset) {
+    def getInvoiceCandidates(String id, String orderNumber, String shipmentNumber) {
         Invoice invoice = Invoice.get(id)
 
         if (!invoice) {
             return []
         }
 
         List<InvoiceCandidate> invoiceCandidates = InvoiceCandidate.createCriteria()
-                .list(max: max.toInteger(), offset: offset.toInteger()) {
-                    if (invoice.party) {
-                        eq("vendor", invoice.party)
-                    }
-
-                    if (invoice.currencyUom?.code) {
-                        eq("currencyCode", invoice.currencyUom.code)
-                    }
-
-                    if (StringUtils.isNotBlank(orderNumber)) {
-                        eq("orderNumber", orderNumber)
-                    }
-
-                    if (StringUtils.isNotBlank(shipmentNumber)) {
-                        eq("shipmentNumber", shipmentNumber)
-                    }
+            .list() {
```

Comment:
I temporarily removed pagination here because I found few issues with API which dragged me away from making it paginated on the front-end (it can be added later, but the performance seems to be very good)

Reply:
Sounds good.